src/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ configure_file (${CMAKE_SOURCE_DIR}/include/fenix-config.h.in
 configure_file(${CMAKE_SOURCE_DIR}/include/fenix.h
   "${CMAKE_BINARY_DIR}/include/fenix.h" COPYONLY)
 
+configure_file(${CMAKE_SOURCE_DIR}/include/fenix.h
+  "${CMAKE_BINARY_DIR}/include/fenix_constants.h" COPYONLY)
+
+configure_file(${CMAKE_SOURCE_DIR}/include/fenix.h
```

Comment:
I think I know why it worked... the compiler actually found a file named `fenix_constants.h` in the include path... (namely, it was an exact copy of `fenix.h`). The thing is that, the symbols actually defined in `fenix_constants.h` (simarly for the other one) were never needed, so the compilation succeeded :)